src/App.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App" />
+  );
```

Comment:
- [ ] Also, create a Book component that takes the book data as a prop and renders the title and the author.

Reply:
- [ ] The Book component should also have a button for removing books, the button wouldn't remove the books yet, this is just a placeholder for now.